content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,64 +14,65 @@
+          3 値状態チェックボックスの一般的な使用法の 1 つは、ソフトウェアインストーラーにあります。そこでは、単一の 3 値状態チェックボックスを使用して、インストールオプションのグループ全体の状態を表し、制御します。
+          そして、グループ内の各オプションは二値状態チェックボックスで個別にオン又はオフにできます。
         </p>
         <ul>
-          <li>If all options in the group are checked, the overall state is represented by the tri-state checkbox displaying as checked.</li>
-          <li>If some of the options in the group are checked, the overall state is represented with the tri-state checkbox displaying as partially checked.</li>
-          <li>If none of the options in the group are checked, the overall state of the group is represented with the tri-state checkbox displaying as not checked.</li>
+          <li>グループ内のすべてのオプションがチェックされている場合、全体の状態は、チェック済みとして表示される3値状態チェックボックスによって表されます。</li>
+          <li>グループ内の一部のオプションがチェックされている場合、全体の状態は、部分的にチェック済みとして表示される3値状態チェックボックスで表されます。</li>
+          <li>グループ内のどのオプションもチェックされていない場合、グループの全体の状態は、チェックされていないと表示される3値状態チェックボックスで表されます。</li>
         </ul>
-        <p>The user can use the tri-state checkbox to change all options in the group with a single action:</p>
+        <p>利用者は、3 値状態チェックボックスを使用して、単一のアクションでグループ内のすべてのオプションを変更できます。</p>
         <ul>
-          <li>Checking the overall checkbox checks all options in the group.</li>
-          <li>Unchecking the overall checkbox will uncheck all options in the group.</li>
+          <li>全体のチェックボックスをチェックすると、グループ内のすべてのオプションがチェックされます。</li>
+          <li>全体のチェックボックスのチェックを外すと、グループ内のすべてのオプションのチェックが外れます。</li>
           <li>
-            And, In some implementations, the system may remember which options were checked the last time the overall status was partially checked.
-            If this feature is provided, activating the overall checkbox a third time recreates that partially checked state where only some options in the group are checked.
+            そして、一部の実装では、システムは、全体の状態が最後に部分的にチェックされたときにどのオプションがチェックされたかを記憶している場合があります。
+            この機能が提供されている場合、全体のチェックボックスを3回目にアクティブにすると、グループ内の一部のオプションのみがチェックされている部分的にチェックされた状態が再作成されます。
           </li>
         </ul>
       </section>
 
       <section id="examples">
         <img alt="" src="../../images/pattern-checkbox.svg">
-        <h2>Examples</h2>
+        <h2>例</h2>
         <ul>
-          <li><a href="examples/checkbox.html">Checkbox (Two-State) Example</a>: Demonstrates a simple 2-state checkbox.</li>
-          <li><a href="examples/checkbox-mixed.html">Checkbox (Mixed-State) Example</a>: Demonstrates a checkbox that uses the mixed value for aria-checked to reflect and control checked states within a group of two-state HTML checkboxes contained in an HTML <code>fieldset</code>.</li>
+          <li><a href="examples/checkbox.html">チェックボックス（二値状態）の例</a>: シンプルな二値状態のチェックボックスを示します。</li>
+          <li><a href="examples/checkbox-mixed.html">チェックボックス（混合状態）の例</a>: HTML <code>fieldset</code> に含まれる二値状態の HTML チェックボックスのグループ内のチェック状態を反映及び制御するために、aria-checked に混合値を使用するチェックボックスを示します。</li>
```

Comment:
ここの「mixed」は属性値の内容を指しているはず

```suggestion
          <li><a href="examples/checkbox-mixed.html">チェックボックス（混合状態）の例</a>: HTML <code>fieldset</code> に含まれる二値状態の HTML チェックボックスのグループ内のチェック状態を反映及び制御するために、aria-checked に mixed を使用するチェックボックスを示します。</li>
```